depthai_nodes/ml/messages/creators/segmentation.py

```diff
@@ -1,61 +1,33 @@
-from typing import List, Union
-
-import depthai as dai
 import numpy as np
 
-from ...messages import SegmentationMasksSAM
+from ...messages import SegmentationMask
 
 
-def create_segmentation_message(x: np.ndarray) -> dai.ImgFrame:
+def create_segmentation_message(mask: np.ndarray) -> SegmentationMask:
     """Create a DepthAI message for segmentation mask.
 
-    @param x: Segmentation map array of the shape (H, W, E{1}) where E{1} stands for the
-        class of the segmented objects.
-    @type x: np.array
-    @return: Output segmentaion message in ImgFrame.Type.RAW8.
-    @rtype: dai.ImgFrame
-    @raise ValueError: If the input is not a numpy array.
-    @raise ValueError: If the input is not 3D.
-    @raise ValueError: If the input 3rd dimension is not E{1}.
+    @param mask: Segmentation map array of shape (H, W) where each value represents a
+        segmented object class.
+    @type mask: np.array
+    @return: Segmentaion mask message.
```

Comment:
missing `t`

Reply:
Fixed in [f4394fd](https://github.com/luxonis/depthai-nodes/pull/127/commits/f4394fd488f0bb1ddb7c87808d4a35ee25dac88a).